discussment-learning/src/main/java/org/danekja/discussment/learning/page/discussion/DiscussionPage.java

```diff
@@ -0,0 +1,80 @@
+package org.danekja.discussment.learning.page.discussion;
+
+import org.apache.wicket.model.IModel;
+import org.apache.wicket.model.Model;
+import org.apache.wicket.request.mapper.parameter.PageParameters;
+import org.danekja.discussment.core.dao.jpa.*;
+import org.danekja.discussment.core.service.*;
+import org.danekja.discussment.core.service.imp.*;
+import org.danekja.discussment.learning.WicketApplication;
+import org.danekja.discussment.learning.page.base.BasePage;
+import org.danekja.discussment.ui.wicket.panel.forum.ForumPanel;
+
+import javax.persistence.EntityManager;
+import java.util.HashMap;
+
+
+
+/**
+ * Homepage
+ */
+public class DiscussionPage extends BasePage {
```

Comment:
Is this page necessary in this example? Discussions are here only as commenting option for articles.